v1/retryable_client.go

```diff
@@ -91,38 +91,31 @@ func (rc *RetryableClient) RetryableBulkImportRelationships(ctx context.Context,
 	})
 
 	_, err = bulkImportClient.CloseAndRecv() // transaction commit happens here
+	if err == nil {
+		return nil
+	}
 
 	// Failure to commit transaction means the stream is closed, so it can't be reused any further
 	// The retry will be done using WriteRelationships instead of BulkImportRelationships
 	// This lets us retry with TOUCH semantics in case of failure due to duplicates
 	retryable := isRetryableError(err)
 	conflict := isAlreadyExistsError(err)
 	canceled, cancelErr := isCanceledError(ctx.Err(), err)
-	unknown := !retryable && !conflict && !canceled && err != nil
 
 	switch {
 	case canceled:
-
 		return cancelErr
-	case unknown:
-
-		return fmt.Errorf("error finalizing write of %d relationships: %w", len(relationships), err)
-	case conflict && conflictStrategy == Skip:
```

Comment:
I wondering about the `skip` scenario. The block is now gone, and I'm wondering what the behaviour will be:
- would it fall under `retryable` and thus would retry when it shouldn't'
- but it fall under `default` because no case catches it?

Reply:
that is a good point. I shouldn't have removed it. It should remain and return nil in this case I believe. I've added that back in